tests/shell/shell_recorder/tutorial_wrapper/Test.md

```diff
@@ -0,0 +1,10 @@
+Test
+```sh
+# Mountpoint:/test
+# testcomment
+$ cat /tmp/test
```

Comment:
the shell command part currently doesn't work because i've had some issues with parsing outputs.
can we add a special character (e.g. `>`) for lines that are expected outputs like
```
kdb get /test/key
> value
```
?
and probably an other special character for stderr too ?